pages/1.11/installing/ent/custom/advanced/index.md

```diff
@@ -191,7 +191,8 @@ In this step you create a custom DC/OS build file on your bootstrap node and the
 
 **Important:**
 
-- Do not install DC/OS until you have these items working: ip-detect script, DNS, and NTP everywhere. For help with troubleshooting, see the [documentation](/1.11/installing/ent/troubleshooting/).
+- Due to cluster configuration issues that caused overlay networks to fail in DC/OS 1.11.0, 1.11.1, and 1.11.2, we currently recommend setting `enable_ipv6` to `false` in `config.yaml` when upgrading _or_ configuring a new cluster. If you have already upgraded to DC/OS 1.11.x without configuring `enable_ipv6` or if `config.yaml` file is set to `true` then do not add new nodes until DC/OS 1.11.3 has been released. You can find additional information and a more robust remediation procedure in our latest critical product advisory.
```

Comment:
Should we change the first sentence to the following?
"Due to a cluster configuration issue with overlay networks, we currently recommend setting `enable_ipv6` to `false` in `config.yaml` when upgrading _or_ configuring a new cluster." 
The phrase "issues that caused overlay networks to fail" seems negative and alarming. But if it's important to emphasize the severity of the issue so customers don't miss it, we can keep the sentence as is. @jenafermiller What's your preference?